Makefile

```diff
@@ -0,0 +1,4 @@
+all: .gitmodules
```

Comment:
Maybe we can avoid the use of makefile by checking presence of  .gitmodules in the script itself?

Reply:
It has global support and I'd rather not reinvent the wheel. It's perfect for this exact use case

Reply:
Got rid of Makefile cc @emrysal